lms/static/sass/views/_program-details.scss

```diff
@@ -102,32 +102,32 @@ $btn-color-primary: palette(primary, dark);
 
 .program-details-wrapper {
   .program-details-header {
-    background-color: $light-gray4;
+    background-color: $gray-100;
     display: flex;
     color: black;
     font-family: 'Open Sans';
     font-weight: normal;
     flex-wrap: wrap;
     padding: 40px 10px 35px;
 
-    @media (min-width: $bp-screen-md) {
+    @include media-breakpoint-up(md) {
       padding: {
         left: 30px;
         right: 30px;
       }
     }
 
-    @media (min-width: map-get($container-max-widths, xl)) {
-      padding: {
-        left: calc(((100% - 1180px) / 2) + 30px);
-        right: calc(((100% - 1180px) / 2) + 30px);
-      }
-    }
+    // @media (min-width: map-get($container-max-widths, xl)) {
```

Comment:
If this is going to be deleted, lets delete it. If it is just a temporary comment, please leave a note saying why it is temporary